packages/node-sdk/README.md

```diff
@@ -138,6 +132,75 @@ The precedence for configuration options is as follows, listed in the order of i
 2. environment variable
 3. the config file
 
+## Type safe feature flags
+
+To get type checked feature flags, add the list of flags to your `bucket.ts` file.
```

Comment:
What's this magical `bucket.ts` file? First mention of it in the README

Since this file isn't required for the consumer to create, let's only mention it as suggestion on how to co-locate bucket specific code

Reply:
recommended creating a bucket.ts further up now

Reply:
I don't like it :/ I get that this is a "recommended" way to organize your code, but the examples becomes a lot harder to read because of it IMO. i'd much rather we provide a single-file example here that can be read linearly from top to bottom. we can add a note where we suggest placing it into a `bucket.ts` singleton file, but that's not required by the lib